src/components/deploymentConfig/DeploymentConfig.tsx

```diff
@@ -31,19 +32,20 @@ export function OptApplicationMetrics({ currentVersion, onChange, opted, focus =
     </div>
 }
 
-export default function DeploymentConfig({ respondOnSuccess }) {
+export default function DeploymentConfig({ respondOnSuccess, isUnSet }) {
     return <div className="form__app-compose">
         <h3 className="form__title form__title--artifatcs">Deployment Template</h3>
         <p className="form__subtitle">Required to execute deployment pipelines for this application.&nbsp;
             <a rel="noreferrer noopener" className="learn-more__href" href={DOCUMENTATION.APP_CREATE_DEPLOYMENT_TEMPLATE} target="_blank">Learn more about Deployment Template Configurations</a>
         </p>
-        <DeploymentConfigForm respondOnSuccess={respondOnSuccess} />
+        <DeploymentConfigForm respondOnSuccess={respondOnSuccess} isUnSet={isUnSet} />
     </div>
 }
 
-function DeploymentConfigForm({ respondOnSuccess }) {
-    const [chartVersions, setChartVersions] = useState<{ id: number, version: string; }[]>(null)
-    const [selectedChart, selectChart] = useState<{ id: number, version: string; }>(null)
+function DeploymentConfigForm({ respondOnSuccess, isUnSet }) {
+    const [chartVersions, setChartVersions] = useState<{ id: number, version: string, name: string; }[]>([])
+    const [selectedChart, selectChart] = useState<{ id: number, version: string, name: string; }>(null)
+    const [selectedChartRefId, setSelectedChartRefId] = useState(0)
```

Comment:
setSelectedChartRefId - this usestate is unused

Reply:
unused deleted

Reply:
fixed, using it now